backend/flow-types/backend-types.js

```diff
@@ -0,0 +1,46 @@
+// @flow
```

Comment:
This file is where we define all custom flow types for our backend. Perhaps they should be defined next to their implementations, but I like the idea of having one file so it is easier to find.